server/models/Films.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const filmsSchema = new Schema({
+  title: String,
+  release_year: Number,
+  actors: [String]
```

Comment:
Are we sure we should leave actors just as a string? In the future we might want to query on other fields relating to them. For instance their age or their previous awards.

We should consider this now because changing it to an actor schema later might be difficult